spec/domain/audits/content_query_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Audits::ContentQuery do
+  let!(:content_item) { create(:content_item) }
+  let!(:audit) { create(:audit, content_item: content_item) }
+
+  it "can filter by audit status" do
+    subject.audit_status(:audited)
+    expect(subject.scope).to eq [content_item]
```

Comment:
The `contain_exactly` matcher would work well here – https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/contain-exactly-matcher

```rb
expect(subject.scope).to contain_exactly(content_item)
```